jsonnet/rules.jsonnet

```diff
@@ -73,6 +73,10 @@
             },
             record: 'cluster:hyperthread_enabled_nodes',
           },
+          {
+            expr: 'count(sum(virt_platform) by (instance, type)) by (type)',
```

Comment:
Can this be simplified to `count(virt_platform) by (type)`?  I'm trying to understand when we'd need to sum first by instance and type, is there a possibility of duplicates?

Reply:
If we don't ever think there will be a) multiple node_exporters per node and b) no chance that these rules are evaluated across multiple scrapers, I would say yes we can.

Reply:
That said, I’m super paranoid about that these days, and wouldn’t want to double count accidentally during node exporter rollouts.  Do you know whether that’s a safe assumption?

Reply:
Yeah, makes sense to be cautious.